### PR TITLE
MAINT: Remove unnecessary backslashes (and replace others by using parenthesis)

### PR DESCRIPTION
**Pull request by MSeifert04**
See also: #13880

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]

**Comment by seberg** [comment]
Can put it in, though while at it, I wouldn't mind just adding the brackets to also remove the "necessary" backslashes in the code that was touched anyway?

**Comment by MSeifert04** [comment]
@seberg Yeah, of course. I didn't remove all backslashes in the code, but I removed those "near" the totally unnecessary backslashes.

**Comment by MSeifert04** [comment]
I wasn't exactly sure how excessive the backslash-removal should be. Let me know if I should remove all the backslashes by introducing brackets in the changed files.

File: numpy/distutils/cpuinfo.py [hunk 1]
```diff
@@ -242,16 +242,16 @@ def _is_Prescott(self):
         return self.is_PentiumIV() and self.has_sse3()
 
     def _is_Nocona(self):
-        return self.is_Intel() \
-               and (self.info[0]['cpu family'] == '6' \
-                    or self.info[0]['cpu family'] == '15' ) \
-               and (self.has_sse3() and not self.has_ssse3())\
-               and re.match(r'.*?\blm\b', self.info[0]['flags']) is not None
+        return (self.is_Intel() \
```
Comment (seberg) [inline numpy/distutils/cpuinfo.py]:
```suggestion
        return (self.is_Intel()
```

**Review by seberg — APPROVED** [review]
Well, feels like if we touch it anyway, but no point of creating too much churn when its obviously a small improvement...

**Comment by charris** [comment]
Thanks @MSeifert04 .

**Pull request merged**